framework/auth/campaigns.py

```diff
@@ -91,6 +91,15 @@ def get_campaigns():
                 }
             })
 
+            newest_campaigns.update({
+                'agu_conference_2023': {
+                    'system_tag': CampaignSourceTags.AGUconference2023.value,
+                    'redirect_url': '',
+                    'confirmation_email_template': mails.CONFIRM_EMAIL_agu_conference_2023,
```

Comment:
```suggestion
                    'system_tag': CampaignSourceTags.AguConference2023.value,
                    'redirect_url': '',
                    'confirmation_email_template': mails.CONFIRM_EMAIL_AGU_CONFERENCE_2023,
```
As per below suggestions

Reply:
These all got overwritten by a force-push, I think, so unresolved